types/address.go

```diff
@@ -256,6 +264,21 @@ func (aa AccAddress) Format(s fmt.State, verb rune) {
 	}
 }
 
+// LengthPrefixAddress prefixes the address bytes with its length, this is used
+// for variable-length components in store keys. Note: All addresses should be
+// max 255 bytes, or else this function panics.
+func LengthPrefixAddress(bz []byte) []byte {
```

Comment:
Let's rename to `LenPrefixedAddress` or `LengthPrefixedAddress` (note _ed_ in prefixed)